Gemfile

```diff
@@ -109,7 +109,7 @@ gem 'statsd-instrument', '~> 2.6.0' # versions beyond 2.6 deprecate config and c
 gem 'swagger-blocks'
 gem 'typhoeus'
 gem 'utf8-cleaner'
-gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: 'master'
+gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', ref: 'bde3b2422f9120790938ac0ea5e031bd0101b9be'
```

Comment:
Is this intentional? The next PR to touch vets-json-schema is likely to change this back to `master`

Reply:
Yes this is intentional.

Just adding the vets-json-schema changes incrementally. If we reference master now, the CI will fail because this code doesn’t support the next “ref” of vets_json_schema yet.